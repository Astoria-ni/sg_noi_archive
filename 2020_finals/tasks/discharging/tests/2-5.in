1484
2737571 3381993 5077998 5146815 5647511 6117227 6193110 7612182 8490561 8858260 10413538 11712348 11972191 12235501 14548629 14839857 14984459 15674295 17075848 18523522 18899552 20926570 21330017 22827622 23672558 25208476 25307543 25361734 25410953 25836764 26509042 29631595 30032090 30822796 32606186 33779461 35322350 35636546 35661427 35771705 36001393 37031646 37286691 39552198 39973797 39984553 40377280 41503708 41703899 44537066 44689787 44772177 44951267 45059446 45884385 47066307 47596357 47654863 49289987 49762233 49841065 50816252 51175142 52037550 52077005 52802520 54272885 54521768 54526200 54698950 56411665 56437651 56609583 57445350 57744588 58320200 58876936 59127047 60469509 60492161 60694704 62194405 64160056 64320826 64820756 64900474 64986137 65183491 65627877 66044609 66292673 66591914 67610645 67662044 67665957 68123705 68736927 69433515 69969962 72654877 72670590 72763970 74249852 74483671 76371322 77708416 77828556 78025171 79458706 79535667 80566886 80879007 82478629 82536820 82737332 83552889 83953612 84259109 85607167 86444406 87305647 87444332 87462133 88120200 89974249 90097721 91340147 91826811 92680171 93311698 93372047 94015520 94217915 94414415 94988227 95587812 95850294 97215376 97503067 97894079 98404802 98834867 101075545 101312149 103737141 104292444 104462944 104561055 105646421 105984006 106351578 107266245 108244075 111676583 112821201 113158559 113326800 113569570 113825244 114705880 114808604 115670941 116241586 116997206 117430756 118603393 118612100 121561946 123015404 123094679 123214819 123484430 123974655 124885298 124885894 125225014 125237743 125580787 125814377 127321938 127578488 128453448 130760638 132848480 133236308 133784846 134774972 134834791 135533893 135640492 135929120 136032639 136224875 137638634 139398473 139415979 140233838 141206342 141879866 142138203 143223196 143608596 144176137 144937391 145640910 145926417 146173089 146476740 146615559 147065557 148544972 148573673 149442922 150465841 151852551 151997328 152045700 153618812 153859935 154871784 155432722 156175542 156189914 156498789 156615501 157948962 161955351 162741562 163003932 163404398 163719798 163809340 164747518 165177485 165827804 167021588 167531282 168592193 168725515 169736034 169944580 171013283 171117920 172897629 174087357 174604401 174723953 175020041 175110950 175417040 175429983 177254955 177629696 178728058 178807287 179015527 179613637 181267219 182719482 182916440 183521509 183743371 184018703 186927777 187321218 187570022 188560266 189854117 190341188 191612074 191774117 192206642 192363087 192383132 193474532 193705442 194093988 194203072 196146083 197182381 198572597 198786275 199538970 199665886 199921050 200227418 200256775 200998998 203204010 203375507 204095272 204186390 205321920 206298878 207529341 209431177 209870338 209943584 213485333 213583134 213770236 214175056 215315932 219010265 219359478 219705050 221517155 221780910 221940645 222231764 222405612 222890719 223224708 223260540 223490649 224172240 224314373 224486513 225439782 226325038 227196187 227967273 229459615 232243946 232258388 232620483 232715657 233210851 234474614 234947896 235119742 235250450 235785845 235831398 237046452 237295311 237717256 237802202 238048263 238230136 238553002 238579777 239003507 239258343 239643424 240132804 240628527 241612523 242199285 242778971 243528089 243959158 244578054 245527827 245597434 245602504 245766092 245766570 245792923 246289473 247260006 247369698 247894627 249282819 250505083 250510383 251160077 253456621 253471113 253719901 254202955 254590091 255147547 255328459 255791737 255915157 256360992 256391285 256572291 258642093 258646401 258825583 259361439 261321278 261603661 261709754 262527741 264766194 264808652 265294667 266769729 266937537 268056238 269100181 269170823 269872580 270240016 270578012 270787841 270828751 273176773 273286391 274259881 274790768 274926280 276220601 277361620 277739599 278291344 279114449 280503573 280599959 281056621 282477605 282605823 283400070 284154354 284254969 284330436 284378687 284916641 284925426 285117525 285796160 285958208 286767805 286825317 289497971 289812843 290951032 291164920 291252316 291399457 292275428 293627529 293631424 294069665 295019005 295051254 295303539 295678530 296268909 297051798 298239331 301021089 301132345 301445859 302240237 303122917 303601569 303956653 304296268 304415488 304686713 305089877 305424850 306267548 307064371 307548603 309760892 310010680 310725627 311835567 312068374 313497335 314105688 315845049 316117363 316265537 316769230 316952109 318418326 318538745 318803144 318936182 318978419 319480949 319902797 319995584 320486166 320723318 321041569 322108723 324696243 324953610 325443577 326368924 327334687 327437323 327488567 330052912 330168124 330195579 330393931 331086480 331174454 331452046 331480743 331949527 332466088 332894800 333122568 334671604 335020256 335377730 335471548 335731158 337057851 337300073 337806496 338520332 339050807 339826682 340390781 343099817 343503179 343829471 344050152 344731889 344793870 345905031 346981716 347053443 348512033 350995455 352128661 352593747 353294137 353395094 353413696 353423634 353486590 353593010 354037988 354413316 357092541 357434154 358231724 358360809 358544221 358813845 358991743 359389767 360299893 360445258 361057699 361479715 362623443 363601989 364025147 364274899 364646607 365027336 365164051 365577883 366332882 367191496 367522946 368157161 368178180 368905849 369014845 369431823 370337463 370816330 371403700 371915233 372245253 372384140 373991585 374616942 374732303 375935317 376012943 376570862 376866968 377003217 377698690 378300129 378927364 379402877 380041651 380167368 380399895 380776465 381073127 381943625 383032610 385777762 385929151 386153941 386564936 387277963 387555964 388133859 388436728 388903018 389219758 389282288 389973394 391423265 391493532 391726913 392078188 392557271 393032520 393374625 395708461 395773388 396501080 397702620 398325635 399336784 400366671 401314740 401457515 402803961 403696418 404692480 405032137 405578485 406569186 408468309 408588409 409698505 410758898 411423288 411690846 412037467 412209396 412502499 412555944 412963520 413977586 414434142 414554043 414684010 414752153 414893956 415089161 415316562 415705854 415903053 416828370 417045415 417530724 418167665 418481091 420216994 420974836 421378329 422395004 422542681 423357207 423426542 424950405 425195022 425359401 425573960 425700622 426666953 426704947 427449518 427848179 428035064 428250690 428568802 429250537 429441601 429456852 429693545 430040393 431413923 432697206 433444379 433474642 433705193 435600822 436181394 436625766 438109609 440037496 441061995 441157629 441735887 443486311 443825670 444388915 444777368 445229588 446033365 446187538 446807850 447317408 447506865 448530243 450767610 451485582 452355215 452834807 452857577 452952069 453047702 453070626 453340036 453653704 454651229 454843622 455386951 455486641 455497412 455662172 455697149 455999730 456407621 456644143 456786073 458125234 458319946 458636131 459560708 460236520 460289553 461260721 461830391 462370412 463087729 463600015 463702215 464262887 464274973 464903853 466707126 467454179 468267785 468471703 469900667 470603523 472111786 472618636 472664621 473923980 473988170 474473353 475003955 475165131 475863805 475896732 477030366 477404058 478702207 479379594 479619083 480435681 481244939 481258833 481831537 482075780 483605108 484432047 484693878 487606211 488098182 488694956 488911308 490365851 490545431 492615295 493363026 493890755 494029146 495893384 498487862 498817154 498862984 500494000 502148804 502477415 503501317 503963714 504738047 504775210 504781724 505882958 506536168 506900881 507869902 507963316 508907212 508922030 510426019 510998185 511938856 513119589 513933143 514844673 515214931 515307520 517653631 517829427 518024670 518420146 521499457 521618908 522798377 523684985 524512887 524908267 525951756 526311492 526502535 527273280 528613778 529399132 530793274 532110860 533139259 534200674 534993590 535268095 536007189 536107867 537121675 538556933 538772718 540598935 540941324 541055516 542802725 543094373 543384727 544022404 544686154 545642686 546693938 547478297 548881667 549707074 549735080 552143943 552313030 552947660 553174721 554134182 554208596 554298901 554435054 555438423 556164007 556341989 556605901 557837120 560659880 560867970 560977198 561426908 563838625 564085906 564256059 564718279 566464065 568527149 568709785 568890248 569876879 570518766 572532504 573519526 573622172 573851181 573854669 574280462 574294197 574653582 574723430 578137095 578164290 578598981 579148166 580646723 581256091 581449053 582373985 584078509 584243952 586319908 586649394 587962243 588130140 588268104 588853429 588910547 589607766 589833608 590554674 590983007 592029815 592788265 592968330 593319260 593354449 593971906 594489039 595124374 595927947 597305568 598163858 598355929 598863038 599281414 601785611 603722621 603970984 605044822 606640269 608550720 610022263 611193570 611233919 612127868 612392187 612483849 613714998 614349092 614366519 614470562 614935133 615674571 615990280 617375517 617985231 619166126 619881966 621997899 622304115 623024783 624971428 625260403 626200543 626733776 627773041 627813232 631369426 633731540 634110600 634243955 634338723 634715390 635200964 635298984 636526928 637462326 637923652 638111497 638846376 639120010 640277076 640478388 641161581 641509054 642702069 642982427 643838338 646975777 647757660 647867084 648759262 649001807 651392150 652296357 652858071 655460543 655510060 655559462 655976852 655996629 656075473 656475311 656613045 656663888 656674683 660204163 661079993 661285930 661547444 661920790 662414204 662788269 662873183 662898980 663618572 664679135 664879102 665031117 665049199 665194682 665410258 665885760 666389592 667213297 667680414 667899931 668006665 668613547 668863187 668905484 669117299 670091958 670525565 672338058 673116486 673542378 674044366 674089499 674357716 674884946 675506106 676680499 677681510 678233824 678405952 680925109 681966637 682479331 682647970 682744390 682797431 682861456 684779816 686288209 686405676 686431918 686451864 686466113 687374102 688467938 688849007 688945264 689566543 690083153 690793255 691355626 691514978 692624619 693009014 694466179 694763457 694789670 694911274 695186627 696421793 696537350 698551932 698644499 699297782 699929345 700891669 701241484 702723698 702895324 703310891 703315663 703517507 704245291 705153607 705279942 706721428 707323300 707620657 707824519 707852915 708026990 711246303 711532796 712440850 713020163 713429471 713700811 715845533 718619447 718969245 719250710 719324330 719966075 720427162 721147208 721883077 722808193 724277787 724308704 724981394 725686812 726147582 726537717 726684307 728791389 730417670 731035306 731652628 731761796 731849717 735543912 736002998 737741396 737975786 738038815 738562427 738819964 740261244 741083837 741671617 742287263 743520236 746163736 747248991 747347748 748468913 748757118 749431169 750618269 751803899 752622793 753465052 754562141 757019277 760157674 760320301 760554677 762677460 763712346 764398220 764760351 765952133 766031001 767988094 768276323 768591980 769777089 770035416 770881087 770901349 771603849 772865076 774576883 774819051 775901909 776922346 777915069 778939697 779347860 781270153 781462440 781589278 782903576 783307638 783395838 783601457 783714914 784073141 785092726 785109595 785823198 786592536 786619417 786667527 787555552 787655049 788261619 789227920 789232796 789619484 790042632 790746101 791725101 791915558 792683736 793896485 794298782 795114427 795219946 796024056 796957591 798763063 800104010 801143129 801484840 801628170 802096897 802238059 802286183 802623298 803311101 805986132 806762373 808111736 809666425 810016915 810874601 811366387 812544076 814480642 817267526 817822987 818201789 818271691 818893062 819039916 819932987 820118599 821119326 823165185 823183387 824636312 825502572 825909402 826022479 826448862 827448252 828822435 829715225 829801553 832309742 833386477 834419461 834580216 835083417 835793695 835909124 835955990 836174979 836269543 836291100 838017455 838047707 838104855 838848369 841566399 841698383 842413855 842607051 842639465 842689062 843708851 843950109 843998631 845288676 845799505 846251935 846551568 846872209 846878356 848685741 850750621 850816241 851877712 851922055 851988712 852836661 854071373 854168900 854747590 856246295 856630387 857168107 857700671 858089531 858828488 859022407 859090495 859330703 860440877 860786350 860789246 860879352 863125696 863560687 864993870 865320800 866406272 867044789 867441052 867788474 869583061 869759081 870772547 871134286 871275156 871399220 871442650 872322707 872981129 873617099 874121448 874866106 874969260 875404644 876181927 876414210 877062235 879095564 880142557 880328490 880931792 882672014 883219695 883988607 884650734 885296664 886056632 886516603 887028772 887281646 888748842 889215976 889261061 889662974 890170113 891300859 891566748 891588061 892872633 893379660 894071820 894502645 894662202 895561630 895892101 896000977 896362274 896710833 896774516 897343479 897603399 898203192 898663578 899337700 899666676 900210526 901105869 902553006 903974938 904481291 905116791 905177479 906101406 907474561 907634901 908238135 908939366 910186673 910409102 912047124 912627296 912765515 914541850 915459914 916171230 916282152 916804242 916825718 917144651 917913941 918239230 919400930 919930846 920178603 920296032 921802944 921808642 922323155 923212127 923261070 923521527 923683637 924058357 924284823 924416898 924437721 925397565 925700906 925796571 927637273 927875147 928132405 928457010 929305967 930998645 931369886 931470594 932837550 932858060 933008333 933284716 933504437 935379120 935620713 935955895 937260304 937836943 937862954 938111759 939254002 939545332 941477316 941634197 941660023 941698072 941920383 941929369 942164632 943094221 944129148 944147368 945167583 946799126 948503069 951509022 951906972 952825719 952864123 952893690 953701153 954144900 954273916 954314484 955394748 955562696 955870337 955931177 955962121 956181496 956200686 959186461 959271070 960845151 961322925 961644477 965889158 966682392 966847100 968073660 968108832 968668379 969172978 969352395 970158881 970251981 970382946 970856628 971143681 971490408 973341387 975154289 975896802 976276003 977833960 978210044 979745598 979988899 980063208 980233745 980568760 981976511 982601939 983076804 983516117 983542687 985629466 989757822 990641707 991856412 991902608 992329744 993824607 993991355 995096499 995173097 996351141 996621954 997086142 997340905 998062451 999337200 999968714
