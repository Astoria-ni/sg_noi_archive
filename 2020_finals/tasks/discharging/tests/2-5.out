1441377743884
